lara-typescript/src/section-authoring/api/api-types.ts

```diff
@@ -230,6 +230,11 @@ export interface ISection {
    * Items in this section
    */
   items?: ISectionItem[];
+
+  /**
+   * Items in this section
```

Comment:
Can you update this comment?

Reply:
Ah, thanks for catching that. It's fixed in the latest commit.